src/renderer/components/input/input.tsx

```diff
@@ -41,7 +41,7 @@ type InputElement = HTMLInputElement | HTMLTextAreaElement;
 type InputElementProps = InputHTMLAttributes<InputElement> & TextareaHTMLAttributes<InputElement> & DOMAttributes<InputElement>;
 
 export type InputProps<T = string> = Omit<InputElementProps, "onChange" | "onSubmit"> & {
-  theme?: "round-black";
+  theme?: "round-black" | "round";
```

Comment:
Did we invent new theme, `round` here?

Reply:
Yes, just like `round-black` isn't a theme that is selectable. (From the `ThemeStore`'s POV).